src/olympia/reviewers/utils.py

```diff
@@ -1017,6 +1018,17 @@ def log_action(
         kwargs = {'user': user or self.user, 'created': timestamp, 'details': details}
         self.log_entry = ActivityLog.objects.create(action, *args, **kwargs)
 
+        attachment = None
+        if self.data.get('attachment_file'):
+            attachment = self.data.get('attachment_file')
+        elif self.data.get('attachment_input'):
+            # The name will be overridden later by attachment_upload_path.
+            attachment = ContentFile(
+                self.data['attachment_input'], name='attachment.txt'
+            )
```

Comment:
So you can't provide both the text _and_ upload a file?

Reply:
It was mentioned in Zoom before that only one file should be attached to each reply -- I don't see where you would need to do this?

I did consider placing a button beforehand to get the user to consciously choose between the two, but uploading a file has more conscious choices (upload -> select file -> submit) where if a file _did_ exist to take, is more likely to be the one the user wants to upload. But I could always just do the button.

Reply:
@diox to clarify/confirm here.

If we _are_ limiting to either the textfield input or an attachment there should be some form validation added to error if both are provided - silently dropping something the reviewer has entered into the textfield isn't great.  (And there could be some better UX, that makes it clear when you provide one of them that you can't provide the other - e.g. disabling fields, etc - but that would be follow-up work)

Reply:
We should not allow both at the same time, and there should be some explicit error about it.